code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -0,0 +1,895 @@
+/mob/living/simple_animal/hostile/guardian
+	name = "Guardian Spirit"
+	real_name = "Guardian Spirit"
+	desc = "A mysterious being that stands by it's charge, ever vigilant."
+	speak_emote = list("intones")
+	response_help  = "passes through"
+	response_disarm = "flails at"
+	response_harm   = "punches"
+	icon = 'icons/mob/mob.dmi'
+	icon_state = "stand"
+	icon_living = "stand"
+	speed = 0
+	a_intent = I_HARM
+	stop_automated_movement = 1
+	floating = 1
+	attack_sound = 'sound/weapons/punch1.ogg'
+	minbodytemp = 0
+	maxbodytemp = 10000000
+	min_oxy = 0
+	max_oxy = INFINITY
+	min_tox = 0
+	max_tox = INFINITY
+	min_co2 = 0
+	max_co2 = INFINITY
+	min_n2  = 0
+	max_n2  = INFINITY
+	attacktext = "punches"
+	maxHealth = 100000 //The spirit itself is invincible
+	health = 100000
+	environment_smash = 0
+	melee_damage_lower = 15
+	melee_damage_upper = 15
+	AIStatus = AI_OFF
+	var/animated_manifest = FALSE
+	var/cooldown = 0
+	var/damage_transfer = 1 //how much damage from each attack we transfer to the owner
+	var/mob/living/summoner
+	var/range = 10 //how far from the user the spirit can be
+	var/playstyle_string = "You are a standard Guardian. You shouldn't exist!"
+	var/magic_fluff_string = " You draw the Coder, symbolizing bugs and errors. This shouldn't happen! Submit a bug report!"
+	var/tech_fluff_string = "BOOT SEQUENCE COMPLETE. ERROR MODULE LOADED. THIS SHOULDN'T HAPPEN. Submit a bug report!"
+	var/bio_fluff_string = "Your scarabs fail to mutate. This shouldn't happen! Submit a bug report!"
+	var/admin_fluff_string = "URK URF!"//the wheels on the bus...
```

Comment:
Should probably add the butcher result of this as being ectoplasm, as well; though it's by no means high priority.